### DISCUSSION TITLE
Dock panels and hide to free body space

### DISCUSSION THREAD

**Discussion by gabfus** [albar965/littlenavmap#815] [Ideas]
Is possible to add in the future a pin to hide and show entire panel in one click. 

This is a sample:
https://docs.telerik.com/devtools/winforms/controls/dock/architecture-and-features/auto-hiding-windows

Thanks for your jobs

**Comment by albar965** [comment]
This would require (even more) workarounds for all the Qt docking window issues.
Actually, the dock window toolbar has the same function, except that the hidden window is not shown in place.
